src/bin/taker.rs

```diff
@@ -82,6 +80,16 @@ enum Commands {
     TotalBalance,
     /// Returns a new address
     GetNewAddress,
+    /// Send to an external wallet address.
+    SendToAddress {
+        /// Recipient address
+        #[clap(name = "address")]
+        address: String,
+        /// Amount to be sent (in satoshis)
+        #[clap(name = "amount")]
+        amount: u64,
```

Comment:
We need a `fee` in this structure itself. That's not the same as the coinswap fee. Then use the fee used here to create the tx.